backend/apps/content/management/commands/seed_lessons.py

```
@@ -292,50 +293,104 @@
+            help="Output format: 'json' for structured data, 'text' for human-readable",
+        )
+
     def handle(self, *args, **options):
+        output_format = options["format"]
+
+        seeded = []
+        skipped = []
+        errors = []
+
         previous_lesson = None
         for l in LESSONS:
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)